magefile.go

```diff
@@ -95,7 +95,7 @@ const (
 	binaryExe  = "fleet-server.exe"
 
 	dockerSuffix      = "main-debian11"
-	dockerArmSuffix   = "base-arm-debian9"
```

Comment:
Not sure why this was `debian9`; It was originally introduced in https://github.com/elastic/fleet-server/pull/4536/, @v1v It looks like we are able to produce binaries with `debian11`. Is there a reason why we should not update?